doc/classes/Label.xml

```diff
@@ -41,8 +41,11 @@
 		</method>
 		<method name="get_total_character_count" qualifiers="const">
 			<return type="int" />
+			<argument index="0" name="include_spaces" type="bool" default="true" />
 			<description>
-				Returns the total number of printable characters in the text (excluding spaces and newlines).
+				Returns the total number of printable characters in the text (excluding newlines).
+				If [code]include_spaces[/code] is set to [code]false[/code], it doesn't count spaces.
+				It's value is true by default.
```

Comment:
```suggestion
				Its value is true by default.
```
But I'd remove this line, the default value is already documented in the prototype.